package.json

```diff
@@ -113,5 +114,7 @@
   "bugs": {
     "url": "https://github.com/companieshouse/node-review-web-starter-ts/issues"
   },
-  "homepage": "https://github.com/companieshouse/node-review-web-starter-ts#readme"
+  "homepage": "https://github.com/companieshouse/node-review-web-starter-ts#readme",
```

Comment:
If we're going to use this, shall we make it the right page: https://github.com/companieshouse/presenter-account-web#readme